django_celery_extensions/task.py

```diff
@@ -465,11 +466,11 @@ def _trigger(self, result, args, kwargs, invocation_id, task_id=None, eta=None,
 
         ignore_task_after_success = (
             ignore_task_after_success if ignore_task_after_success is not None
-            else self.ignore_task_after_success_timedelta is not None
+            else self.ignore_task_timedelta is not None
```

Comment:
Shouldn't this be `self.ignore_task_after_success`?

Reply:
No this must work for manual and automatic (after success) ignore